study-builder/studies/circadia/study.conf

```diff
@@ -0,0 +1,672 @@
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "HEALTH_AND_LIFESTYLE_QUESTIONNAIRE"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": """user.studies["circadia"].forms["HEALTH_AND_LIFESTYLE_QUESTIONNAIRE"].isStatus("COMPLETE")"""
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "DASHBOARD"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "CONSENT",
+          "expression": """user.studies["circadia"].forms["CONSENT"].isStatus("CREATED", "IN_PROGRESS")"""
+        },
+        {
+          "type": "ACTIVITY",
+          "activityCode": "DLMO_SCHEDULING_INSTRUCTIONS",
+          "expression": """user.studies["circadia"].forms["DLMO_SCHEDULING_INSTRUCTIONS"].hasInstance()"""
+        },
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "MINI_SLEEP_LOG"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "DLMO",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY"
+        "activityCode": "DLMO",
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "POST_COLLECTION_QUESTIONNAIRE",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "POST_COLLECTION_QUESTIONNAIRE"
+      },
+      "to": [
+        {
+          "type": "ACTIVITY",
+          "activityCode": "KIT_RETURN_ATTESTATION",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "KIT_RETURN_ATTESTATION"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "POST_COLLECTION_ATTESTATION_1"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "POST_COLLECTION_ATTESTATION_2"
+      },
+      "to": [
+        {
+          "type": "DASHBOARD",
+          "expression": "true"
+        }
+      ]
+    },
+  ],
+
+  "events": [
+
+
+    # activity instance creation events
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "PREQUAL",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": "CONSENT"
+      },
+      "preconditionExpr": """
+      (
+(
+!user.studies["circadia"].forms["PREQUAL"].questions["ATYPICAL_CIRCADIAN_RHYTHM"].answers.hasOption("ACR_NO")
```

Comment:
Option stable ids only need to be unique within the question itself. So it's okay to have option ids like `YES` and `NO` for multiple questions. You don't have to namespace your option ids with `ACR_`, `CRDS_`, etc. I recommend shortening the option ids -- it'll probably help make these PEX expressions shorter.

Reply:
@yufengwng Oh, I didn't know about that, thanks!